lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb

```diff
@@ -6,60 +6,23 @@ module Fastlane
 
   module Helper
     module FirebaseAppDistributionHelper
-      def testers_flag(params)
-        file_flag_if_supplied("--testers-file", "testers", params)
-      end
-
-      def groups_flag(params)
-        file_flag_if_supplied("--groups-file", "groups", params)
-      end
-
-      def release_notes_flag(params)
-        file_flag_if_supplied("--release-notes-file", "release_notes", params)
-      end
-
-      def file_flag_if_supplied(flag, param_name, params)
-        file = params["#{param_name}_file".to_sym]
-        file ||= file_for_contents(param_name.to_sym, params)
-
-        if file
-          return "#{flag} #{file}"
+      def get_value_from_value_or_file(value, path)
+        if (value.nil? || value.empty?) && !path.nil?
+          begin
+            return File.open(path).read
+          rescue
+            UI.crash!("#{ErrorMessage::INVALID_PATH}: #{path}")
+          end
         end
+        value
       end
 
-      def flag_value_if_supplied(flag, param_name, params)
-        "#{flag} #{params[param_name]}" if params[param_name]
-      end
-
-      def flag_if_supplied(flag, param_name, params)
-        flag if params[param_name]
-      end
-
-      ##
-      # always return a file for a given content
-      def file_for_contents(parameter_name, params)
-        if @tempfiles.nil?
-          @tempfiles = []
+      def string_to_array(string)
+        if string.nil?
+          return string
         end
-
-        contents = params[parameter_name]
-        return nil if contents.nil?
-
-        file = Tempfile.new(parameter_name.to_s)
-        file.write(contents)
-        file.close
-        @tempfiles << file
-
-        file.path
-      end
-
-      def cleanup_tempfiles
-        return if @tempfiles.nil?
-        @tempfiles.each(&:unlink)
-      end
-
-      def parse_plist(path)
-        CFPropertyList.native_types(CFPropertyList::List.new(file: path).value)
+        string_array = string.gsub(/\s+/, '').split(",")
```

Comment:
We do not need to remove whitespace, but felt that it gave the customer more flexibility in terms of layout. Now they can have a string that is something like "Manny , Manny2 " rather than "Manny,Manny2" for readability. 

This method works with strings that have whitespace and those that don't. Also if we give the API a group or tester with a white space we will get a 400 status code.

Reply:
> We do not need to remove whitespace, but felt that it gave the customer more flexibility in terms of layout. Now they can have a string that is something like "Manny , Manny2 " rather than "Manny,Manny2" for readability.

That makes sense 👍 

> This method works with strings that have whitespace and those that don't. Also if we give the API a group or tester with a white space we will get a 400 status code.

That's true, but as a reader if I look at the method name and the description I might try to use this in another spot to separate a comma-separated list of values where the values themselves might have spaces in them, like: "First Value, Second Value". And it wouldn't work for that. I think that's fine but you should make it clear to the reader, maybe in the method description.

Reply:
That makes sense. I have added in an additional comment with an example hopefully to clarify that.

Reply:
Looks great 👍